src/main/tabs/playing.js

```diff
@@ -1002,8 +1003,15 @@ Main.k.tabs.playing = function() {
 	/**
 	 * @return string;
 	 */
-	Main.k.FormatPlanets = function(index) {//TODO: MULTILANG
-		var ret = "**//"+Main.k.text.gettext("Planètes")+" : //**";
+	Main.k.FormatPlanets = function(index,tooltip) {//TODO: MULTILANG
+		var ret = '',tooltipret = '',nbrPlanet = $("#navModule").find(".planet").not(".planetoff").length, compteurPlanet = 0;
```

Comment:
Petit soucis d'alignement et pour plus de lisibilité vaut mieux mettre chaque init de variable sur sa propre ligne.
Renommer `tooltipret` en `tooltipRet` (je n'ai pas fais, y en a trop ;)) 
```suggestion
	var ret = '';
	var tooltipret = '';
	var nbrPlanet = $("#navModule").find(".planet").not(".planetoff").length;
	var compteurPlanet = 0;
```